### ISSUE TITLE
morphTo() could be called without arguements

### ISSUE THREAD

**Issue by szepeviktor** [nunomaduro/larastan#425]
https://github.com/nunomaduro/larastan/blob/master/src/Types/RelationDynamicMethodReturnTypeExtension.php#L53-L54

causes

```
PHP Fatal error:  Uncaught TypeError: Argument 1 passed to NunoMaduro\Larastan\Analyser\Scope::getType() must be an instance of PhpParser\Node\Expr, null given, 
called in vendor/nunomaduro/larastan/src/Types/RelationDynamicMethodReturnTypeExtension.php on line 54 and 
defined in vendor/nunomaduro/larastan/src/Analyser/Scope.php:42
```

https://github.com/laravel/framework/blob/6.x/src/Illuminate/Database/Eloquent/Concerns/HasRelationships.php#L222

**Comment by szepeviktor** [comment]
Lazy bug report!

**Comment by canvural** [comment]
Thank you! Good catch.

Fixed in #426
